apps/fail2ban.sh

```diff
@@ -184,4 +186,46 @@ msg_box "Fail2ban is now successfully installed.
 Please use 'fail2ban-client set nextcloud unbanip <Banned IP>' to unban certain IPs
 You can also use 'iptables -L -n' to check which IPs that are banned"
 
+# Daily ban notification
+if ! yesno_box_no "Do you want to get notified about daily bans?\n
+If you choose 'yes', you will receive a notification about daily bans at 23:59h."
+then
+  exit
+fi
+
+# Create Fail2ban report script
+cat << FAIL2BAN_REPORT > "$SCRIPTS/daily_fail2ban_report.sh"
+#!/bin/bash
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# Look for ip addresses
+BANNED_IPS=\$(grep "Ban " /var/log/fail2ban.log | grep "\$(date +%Y-%m-%d)" \
+| awk -F "NOTICE  " '{print "Jail:",\$2}' | sort)
+
+# Exit if nothing was found
+if [ -z "\$BANNED_IPS" ]
+then
+    exit
+fi
+
+# Report if something was found
+source /var/scripts/fetch_lib.sh || source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+notify_admin_gui \
+"Your daily fail2ban report" \
+"Those got banned today:
```

Comment:
Please change to: These IP's got banned today:

Reply:
Done